File: java/lc1455.java
class Solution {
    public int isPrefixOfWord(String sentence, String searchWord) {
        String[] words = sentence.split(" ");
        int idx = -1;
        int cur = 1;
        for(String s: words)
        {
            if(s.startsWith(searchWord))
            {
                idx = cur;
                break;
            }
            cur++;
        }
        return idx;   
    }
}

File: cs/lc1455.cs
public class Solution {
    public int IsPrefixOfWord(string sentence, string searchWord) {
        string[] words = sentence.Split(" ");
        int idx = -1;
        int cur = 1;
        foreach(string s in words)
        {
            if(s.StartsWith(searchWord))
            {
                idx = cur;
                break;
            }
            cur++;
        }
        return idx;            
    }
}
